v2/jdbc-to-googlecloud/src/test/java/com/google/cloud/teleport/v2/templates/JdbcToBigQueryIT.java

```diff
@@ -201,48 +203,19 @@ public void testMySqlToBigQueryWithStorageWriteApi() throws IOException {
   }
 
   @Test
-  public void testPostgresToBigQueryFlex() throws IOException {
```

Comment:
testPostgresToBigQueryFlex and testPostgresWithUnicodeCharactersInQuery mostly duplicates. Consolidate into one. There is a still a third postgres integration test "testPostgresReadWithPartitions"